data/processors/maps.py

```diff
@@ -128,6 +130,9 @@ def assign_roomfinder_maps(data):
     # Read the Roomfinder maps
     with open("external/maps_roomfinder.json") as f:
         maps_list = json.load(f)
+        
+    # There are also Roomfinder-like custom maps, that we assign here
+    custom_maps = _load_custom_maps()
```

Comment:
I dont love, how this is 51 lines away from its first usage.
This is however not a thing I'd expect you to fix, since this method would be refactored for this to be fixed. (that would have to be another PR)

Reply:
The whole functions are quite large actually, which is certainly not ideal. The reason I put this there is becaue I wanted to group map loading code.

Reply:
I know, that is something I would have done too, but we should keep this in mind, before adding to this function in the future